config/database.yml

```diff
@@ -1,84 +1,16 @@
-# PostgreSQL. Versions 9.1 and up are supported.
-#
-# Install the pg driver:
-#   gem install pg
-# On OS X with Homebrew:
-#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
-# On OS X with MacPorts:
-#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
-# On Windows:
-#   gem install pg
-#       Choose the win32 build.
-#       Install PostgreSQL and put its /bin directory on your path.
-#
-# Configure Using Gemfile
-# gem 'pg'
-#
 default: &default
   adapter: postgresql
   encoding: unicode
-  # For details on connection pooling, see rails configuration guide
-  # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
 
 development:
   <<: *default
-  database: michigan_prototype_development
+  database: <%= ENV["DATABASE_URL"] || 'michigan_prototype_development' %>
```

Comment:
I get a little anxious putting `DATABASE_URL` here as the default. It would take just one time that someone forgets to unset that ENV and something catastrophic happens to the production database. I think running against the prod database should be a little painful, or better put -- a bit of a "chore".  EG: having to change `database.yml` before running, making sure to revert it before committing anything, etc.